agent/docker-compose.yml

```
@@ -0,0 +1,16 @@
+services:
+  glouton:
+    image: bleemeo/glouton
+    restart: unless-stopped
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /:/hostroot:ro
+      - ./glouton.conf:/etc/glouton/conf.d/90-mqtt.conf:ro
+      - glouton-data:/var/lib/glouton
+    network_mode: host
+    pid: "host"
+    cap_add:
+      - SYS_PTRACE # This permission is needed to monitor processes IO and memory used per process.
+      - SYS_ADMIN # This permission is needed to retrieve the network interface names of containers.
+    depends_on:
+      - nats
```

```suggestion

volumes:
  glouton-data: {}
```

NATS is not defined in this Docker compose and the volume for Glouton is not defined in this Docker compose.